SolidBeginnerWorkshopNotes.md

```diff
@@ -0,0 +1,149 @@
+        implementing from specifications!)
+    - <https://solidproject.org/users/get-a-pod>
+        - Many Pod Providers to choose from! Some community-based,
+            some by companies.
+        - A Pod Provider will usually advertise the underlying cloud
+            infrastructure it is using, and the hosting location. For
+            example, Inrupt Pod Spaces uses Amazon for hosting and the
+            hosting location is Germany.
+
+- **Solid applications:**
+    - Any type of application can be built using the Solid Protocol.
+    - Applications can make requests to retrieve data from your Pod (via your Pod Provider), and also to store data on your Pod. This is the essential part of how Solid works. 
+        - Keep in mind that you ALWAYS control access of your files on your Pod, so you can revoke access to an application at any time. 
+    - Example of applications: <https://solidproject.org/apps>
+    - Example active funded projects:
+        - health care use case (name?) by Marc Haddle
+        - ???
+
+## Solid in context
+
+- Solid is a specification consisting of several Technical Reports,
+    each edited by a Solid panel.
+    - Anyone can contribute by [filing an issue on
+        GitHub](https://github.com/solid/specification/issues)
+    - Technical Reports: <https://solidproject.org/TR/> include the
+        [Solid Protocol](https://solidproject.org/TR/protocol), [Solid
+        WebID Profile](https://solid.github.io/webid-profile/), [Web
+        Access Control](https://solidproject.org/TR/wac), [Shape
+        Trees](https://shapetrees.org/TR/specification/), [Solid
+        Chat](https://solid.github.io/chat/), and lots more!
+
+- The authors of the technical reports are the [Solid Community
+    Group](https://www.w3.org/groups/cg/solid/) (CG) of the
+    [W3C](https://www.w3.org/).
+    - W3C is the World Wide Web Consortium: they *develop standards
+        and guidelines to help everyone build a web based on the
+        principles of accessibility, internationalization, privacy and
+        security.*
+
+- Solid has a new organizational home, the Open Data Institute (ODI),
+    as announced by Sir Tim Berners-Lee on October 2024. [Link to
+    post](https://forum.solidproject.org/t/a-new-organisational-home-for-solid/8004)
+    - The ODI is a non-profit company that aims to build a world where
+        data works for everyone. It was founded in 2012 by Sir Tim
+        Berners-Lee and Sir Nigel Shadbolt.
+
+- Solid community channels
+    - Solid Forum: <https://forum.solidproject.org/>
+    - Matrix channels:
+        [https://matrix.to/#/#solid_project:matrix.org](https://matrix.to/#/#solid_project:matrix.org)
+    - Solid Practitioners Bi-Weekly meeting:
+        - https://www.w3.org/events/meetings/ba3d6036-ef79-42f1-acc7-8ce9d3bc00ba/20271007T150000/
+        - <https://www.w3.org/events/meetings/b2bcc109-a7e7-4e9c-ba9f-8dcd234e5275/20270916T150000/>
```

Comment:
If you're going to point to meetings then also:
 - CG meeting https://www.w3.org/community/solid/
 - SolidOS meetings https://github.com/SolidOS/solidos for those with some experience with Solid that want to contribute to SolidOS